core/src/main/java/org/geysermc/geyser/network/netty/GeyserServer.java

```diff
@@ -94,19 +95,27 @@ public final class GeyserServer {
 
     private final GeyserImpl geyser;
     private EventLoopGroup group;
+    // Split childGroup may improve IO
+    private EventLoopGroup childGroup;
     private final ServerBootstrap bootstrap;
     private EventLoopGroup playerGroup;
 
     @Getter
     private final ExpiringMap<InetSocketAddress, InetSocketAddress> proxiedAddresses;
+    private final int listenCount;
 
-    private ChannelFuture bootstrapFuture;
+    private ChannelFuture[] bootstrapFutures;
 
     public GeyserServer(GeyserImpl geyser, int threadCount) {
         this.geyser = geyser;
-        this.group = TRANSPORT.eventLoopGroupFactory().apply(threadCount);
+        int defaultListenCount = 4; // 4 should be a pretty perfect value, as it is unlikely to cause disconnections and also won't waste any performance.
```

Comment:
Personally, I'd set the default value to 2 - the average server with Geyser doesn't have more than ~5 Bedrock players online anyways

Reply:
Understood, let's set it to 2 !